docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -28,6 +31,14 @@ const RadioButtonExampleSimple = () => (
         style={styles.radioButton}
       />
     </RadioButtonGroup>
+
+    <RadioButtonGroup name="notRight" labelPosition="left" style={styles.block}>
```

Comment:
Because `labelPosition` is a `RadioButtonGroup` property. I could put in a divider and additional `RadioButton` if you felt it needed to be more explicit in the example in addition to in the code. It is mentioned in the description already.